chadwick_bissette/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "chadwick_bissette",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "chai": "^3.4.0"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
```

Comment:
Put your email in here so people who download your package know who to contact with bugs